Tests/Functional/Hooks/class.tx_multicolumn_tcemainTest.php

```diff
@@ -25,10 +25,14 @@
 
 use TYPO3\CMS\Core\Core\Bootstrap;
 use TYPO3\CMS\Core\DataHandling\DataHandler;
-use TYPO3\CMS\Core\Tests\FunctionalTestCase;
+use TYPO3\CMS\Core\Tests\Functional\DataHandling\AbstractDataHandlerActionTestCase;
 
-class tx_multicolumn_tcemainTest extends FunctionalTestCase
+class tx_multicolumn_tcemainTest extends AbstractDataHandlerActionTestCase
```

Comment:
Please do not use the DataHandler TestCase. It is an internal abstraction which is not needed for the tests in this extension. We don't want to test the DataHandler but the hook which are called from it.